platforms/native/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/msvcpp/VisualCppPlatformToolProvider.java

```diff
@@ -248,7 +248,7 @@ public WindowsSdkLibraries getSystemLibraries(ToolType compilerType) {
         return libraries;
     }
 
-    private <T extends NativeCompileSpec> Transformer<T, T> addDefinitions(Class<T> type) {
+    private <T extends NativeCompileSpec> Transformer<T, T> addDefinitions(@SuppressWarnings("UnusedVariable") Class<T> type) {
```

Comment:
❔ Did you find that the `type` variable is still required or is removal too cumbersome? It might be worth it to have a comment/TODO describing the situation for the next person to stumble upon this.

Reply:
right I could just remove it.